### ISSUE TITLE
Pull request 558 changed initialisation behaviour

### ISSUE THREAD

**Issue by gsobala** [LeelaChessZero/lc0#566]
Pull request 558 changed the initialisation behaviour after uci ``isready``. It was stated in the discussion that "I checked a few more logs, every one of them always sends position right before go so doing the initialization in go (as we do now as a last resort) is at worst a few ns late."

Actually it can be much more than a few ns as this logfile from an initialisation on a cloud V100 shows. This one took about 11 seconds (quite usual for these cloud instances). This will impact matches played under the Chessbase family of GUIs and also Aquarium I think, i.e. any GUI that does not send ``ucinewgame``.
```
============= Log started. =============
1207 07:59:55.310030 140531400898368 ../src/main.cc:36] Lc0 started.
1207 07:59:55.310095 140531400898368 ../src/main.cc:37]        _
1207 07:59:55.310297 140531400898368 ../src/main.cc:38] |   _ | |
1207 07:59:55.310402 140531400898368 ../src/main.cc:39] |_ |_ |_| v0.20.0-dev built Dec  7 2018
1207 07:59:55.310504 140531400898368 ../src/utils/commandline.cc:45] Command line: ./lc9 --logfile=lc9.log
1207 07:59:59.661583 140531400898368 ../src/chess/uciloop.cc:131] >> isready
1207 07:59:59.661687 140531400898368 ../src/chess/uciloop.cc:218] << readyok
1207 08:00:07.524510 140531400898368 ../src/chess/uciloop.cc:131] >> go movetime 60000
1207 08:00:07.599992 140531400898368 ../src/neural/loader.cc:181] Found pb network file: ./11250.pb.gz
1207 08:00:08.414632 140531400898368 ../src/neural/factory.cc:84] Creating backend [cudnn]...
1207 08:00:19.525429 140531400898368 ../src/engine.cc:378] Limits: visits:4000000000 playouts:-1 depth:-1 infinite:
0 search_deadline:1207 08:00:59.461684
1207 08:00:19.525475 140531400898368 ../src/engine.cc:398] Timer started at 1207 07:59:59.661684
1207 08:00:19.525579 140530931341056 ../src/mcts/search.cc:589] Start a watchdog thread.
1207 08:00:19.525878 140530669389568 ../src/mcts/search.h:200] Started search thread.
1207 08:00:19.526136 140530677782272 ../src/mcts/search.h:200] Started search thread.
1207 08:00:19.580923 140530669389568 ../src/chess/uciloop.cc:218] << info depth 1 seldepth 2 time 55 nodes 6 score 
cp 20 hashfull 0 nps 109 tbhits 0 pv d2d4 g8f6
```

**Comment by borg323** [comment]
What was the uci host in this test? I notice there is no `position` send to lc0 in this log excerpt.
The quoted phrase was about the time delay between `position` and `go`, so not applicable here.

Note that the timer started counting at 07:59:59.661684 (1 ns after the `isready` command) so the initialization time is taken out of the first move, as expected in such cases.

**Comment by gsobala** [comment]
The host was a human

**Comment by Naphthalin** [comment]
Should be resolved with #1216 being merged.
